client/jetpack-connect/plans-skip-button.jsx

```diff
@@ -9,11 +9,13 @@ import { localize } from 'i18n-calypso';
  * Internal dependencies
  */
 import Button from 'components/button';
+import Gridicon from 'gridicons';
 
 const PlansSkipButton = ( { onClick, translate } ) => (
 	<div className="jetpack-connect__plans-nav-buttons">
-		<Button onClick={ onClick } compact borderless>
-			{ translate( 'Skip' ) }
+		<Button onClick={ onClick }>
+			{ translate( 'Start with free' ) }
+			<Gridicon icon="arrow-right" />
```

Comment:
We'll need to flip this in RTL context

Reply:
whoa nice catch

Reply:
Indeed awesome catch Jon! We can use the `isRtl()` selector for determining the right `icon`, RTL compatibility in the stylesheet will be taken care of automatically by `rtlcss`.